common/js/options.js

```diff
@@ -110,7 +115,48 @@ document.addEventListener("DOMContentLoaded", function() {
         // only run the following when on options.htm
         return;
     }
-
+	
+	function handleSizeChange() {
+		if (bodyHeight) {
+			preview.style.height = bodyHeight.value + "px";
+		}
+		if (bodyWidth) {
+			preview.style.width = bodyWidth.value + "px";
+		}
+	}
+	if (bodyWidth) {
```

Comment:
There were some issues where bodyWidth didn't exist, so it now checks for that before doing anything.